### PR TITLE
Update public_transport funk

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to #469

@nvkelso, @zerebubuth could you review please?

### CODE REVIEW

**Comment by nvkelso** [comment]
LGTM

File: spreadsheets/kind/transit.csv [hunk 1]
```diff
@@ -6,9 +6,9 @@ highway,public_transport,railway,route,service,tags->rail,tags->site,min_zoom,ki
 *,*,*,funicular;monorail,*,*,*,${12},${route}
 *,*,halt;stop;tram_stop,*,*,*,*,${13},${railway}
 platform,*,*,*,*,*,*,${15},platform
-*,platform,*,*,*,yes,*,${15},platform
+*,platform,*,*,*,yes,*,${17},platform
```
Comment (zerebubuth) [inline spreadsheets/kind/transit.csv]: Should the kind here be `bus_stop`?

**Comment by zerebubuth** [comment]
I think the changes to POIs would make it turn something that was plain `public_transport=platform` into a railway platform, where it's missing the positive `tags->bus='yes'`? Should that be `tags->rail != 'yes'` instead?

Also, tests would be good for figuring out what the effect of these would be, i.e: including one for the various different [common cases](https://github.com/mapzen/vector-datasource/issues/469#issuecomment-199357595).

File: spreadsheets/kind/transit.csv (hunk 1 above)
Comment (rmarianski) [inline spreadsheets/kind/transit.csv]: Nice catch, fixed in 08b3202.

File: spreadsheets/kind/transit.csv [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-*,*,*,*,*,*,stop_area,${15},stop_area
-bus_stop,*,*,*,*,*,*,${17},bus_stop
+highway,public_transport,railway,route,service,tags->light_rail,tags->rail,tags->site,min_zoom,kind
+*,*,*,train,high_speed;long_distance;international,*,*,*,${5},${route}
+*,*,*,train,*,*,*,*,${6},train
+*,*,*,subway,*,*,*,*,${8},subway
+*,*,*,light_rail;tram,*,*,*,*,${9},${route}
+*,*,*,funicular;monorail,*,*,*,*,${12},${route}
+*,*,halt;stop;tram_stop,*,*,*,*,*,${13},${railway}
+platform,*,*,*,*,*,*,*,${15},platform
```
Comment (zerebubuth) [inline spreadsheets/kind/transit.csv]: Shouldn't `highway=platform` also be a `bus_stop` at `z >= 17`?

Comment (rmarianski) [inline spreadsheets/kind/transit.csv]: Updated in 01cce9a.

**Comment by rmarianski** [comment]
@zerebubuth I added a few tests, do you think it looks good enough to merge now?

**Comment by nvkelso** [comment]
Looking good. Let's get this on dev.

**Comment by rmarianski** [comment]
I'll merge this in, we can make any adjustments afterwards if need be.

**Pull request merged**